### ISSUE TITLE
 solicit::http::HttpError::Other(Box<Error>)` is not `Send + Sync`

### ISSUE THREAD

**Issue by jamii** [mlalic/solicit#6]
This percolates up through hyper and breaks rust-websocket. You can probably just add `Send + Sync` to the type, and you can assert that the Error is `Send + Sync` by adding something like:

``` rust
fn _assert_error_is_sync_send() {
    fn _is_sync_send<T: Sync+Send>() {}
    _is_sync_send::<Error>();
}
```

**Comment by mlalic** [comment]
Thanks for the report!

Should be fixed (and the regression test included) in 2f6f449.

I'll merge it and release a new version on crates.io tomorrow during the day (CEST).

**Comment by jamii** [comment]
Awesome, thanks :)
